### ISSUE TITLE
Cant download Songs.

### ISSUE THREAD

**Issue by Ascarva** [kihecido/BMBF#75]
Hey, i installed the new Verion of BMBF 1hour ago. 
I uninstalled the old version of BMBF and BeatSaber with Sidequest.
New BMBF installed fine.

But now i have 2 issues.

The BSaber Login in the Headset via BMBF doesnt work. It accepts my ID and Password but im still not logged in.

It dont download my Bookmarks (I dont know if i have to be logged in for that). But it doesnt download any other song either.

**Comment by markhicken** [comment]
People. You really need to look at existing issues before creating new ones.

**Comment by KutaroV2** [comment]
Yeah I'm having the same issue. Won't login and always says 'download failed' when I try to install songs.

**Comment by Brittz69** [comment]
I also have the same problem downloading song, uploaded all old custom tracks fine and if I have bought official tracks I can download them in game fine.
But can't install new custom tracks and tried to download an official in game track and pay for it, this also failed. 
Hope this helps

**Comment by Ldude162** [comment]
I have all of the problems the first person has

**Comment by SoySauske7** [comment]
same

**Comment by LiTLNugget** [comment]
I almost have the same problem. Everything's fine but I can't download songs. I've uninstalled BS and BMBF and reinstalled them and yet I still can't download any songs even when I have my previous tracks downloaded and with the latest update on BS
